src/worldcereal/openeo/feature_extractor.py

```diff
@@ -0,0 +1,115 @@
+"""Feature computer GFMAP compatible to compute Presto embeddings."""
+
+import xarray as xr
+from openeo_gfmap.features.feature_extractor import PatchFeatureExtractor
+
+class PrestoFeatureExtractor(PatchFeatureExtractor):
+    """Feature extractor to use Presto model to compute embeddings.
+    This will generate a datacube with 128 bands, each band representing a
+    feature from the Presto model.
+    """
+
+    import functools
+    from pathlib import Path
+    from typing import Tuple
+
+    PRESTO_PATH = "https://artifactory.vgt.vito.be/artifactory/auxdata-public/worldcereal-minimal-inference/presto.pt"  # NOQA
+    BASE_URL = "https://s3.waw3-1.cloudferro.com/swift/v1/project_dependencies"  # NOQA
+    DEPENDENCY_NAME = "wc_presto_onnx_dependencies.zip"
+
+    GFMAP_BAND_MAPPING = {
+        "S2-L2A-B02": "B02",
+        "S2-L2A-B03": "B03",
+        "S2-L2A-B04": "B04",
+        "S2-L2A-B05": "B05",
+        "S2-L2A-B06": "B06",
+        "S2-L2A-B07": "B07",
+        "S2-L2A-B08": "B08",
+        "S2-L2A-B8A": "B8A",
+        "S2-L2A-B11": "B11",
+        "S2-L2A-B12": "B12",
+        "S1-SIGMA0-VH": "VH",
+        "S1-SIGMA0-VV": "VV",
+        "COP-DEM": "DEM",
+        "A5-tmean": "temperature-mean",
```

Comment:
I think we haven't discuss these band names yet. I get the temptation of sticking always to 2 character 'sensor' specification but `A5` really doesn't say much and often in the future it won't be possible in a good way to stick to two characters anyway. I propose to use the full `AGERA5` specification in the GFMAP band names. Shouldn't we also stick to uppercase? `TMEAN` in that case?